Chap_Introduction.tex

```diff
@@ -8,24 +8,24 @@ \chapter{Introduction}
 \label{chap:intro}
 
 \added{ \ac{PMIx} is an application programming interface specification to provide
-High Performance Computing libraries and programming models with
-portable and well defined access to commonly needed
-distributed computing system software.
-The \ac{PMIx} community strives to define interfaces to support the most widely
-used HPC programming models and libraries.
-\ac{PMIx} gives distributed systems software providers a better understanding of what
-services can be provided to best support these programming models.
-It also encourages the implementation of \ac{PMIx} APIs which allow for
+libraries and programming models with a portable and well-defined access to commonly
+needed services in distributed and parallel computing systems.
+A typical example of such a service is the exchange of information to start a \ac{HPC} service
+or application in a portable and scalable manner.
+As such, \ac{PMIx} also gives distributed system software providers a better understanding of how
+programming models and libraries can interface with and use system-level services.
+As a specification, \ac{PMIx} also promotes the use and implementation of APIs that allow for
 portable access to these varied system software services and the
-functionalities they offer.   Although these APIs can be implemented directly by the
+functionalities they offer.
+Although these APIs can be defined and implemented directly by the
 system software components providing them, the \ac{PMIx} community
-feels the development of centralized \ac{PMIx} implementations better serves the
-HPC community.  Such implementations can interface
-with existing system software or directly implement PMIx API's which are
-not already provided by existing system software components.
-\ac{PMIx} allows programming languages and libraries to focus on their core
-competencies without having to provide their own services to provide
-services that are better provided at a system level. }
+feels that the development of shared specification better serves the
+community.
```

Comment:
a shared specification

Reply:
a standard?

Reply:
Is there a better term than PMIx  community?  Do we already can the document creators by a different name somewhere in the doc?